sdk/core/perf/src/program.cpp

```diff
@@ -320,6 +320,13 @@ void Azure::Perf::Program::Run(
       // Take the corresponding proxy from the list in round robin
       parallelTest[i]->SetTestProxy(options.TestProxies[i % options.TestProxies.size()]);
     }
+    if (options.Insecure)
+    {
+#if defined(BUILD_TRANSPORT_WINHTTP_ADAPTER)
+      std::cout << std::endl << " ERROR !!! winHTTP does not support using insecure mode.";
```

Comment:
Normally when printing messages, the crlf goes at the end of the message, not the beginning. Otherwise the next line of output gets messed up.

Reply:
Pretty much every "hello world" app that I've ever seen has had `printf("hello world!\n");` I don't think I've ever seen a hello world with `printf("\nhello world!")`.

At a minimum, I find the whole `foolarryo@LOTHAIR-HOME:/mnt/g/Az/LarryO/azure-sdk-for-cpp$ ` behavior that results from not having the `endl` be at the end of the line.

Reply:
Btw, note that in this case, the lack of a `std::endl` at the end of the line is limited because line 333 has a `std::endl` before the `=== Global Setup ===`. Looking at line 333, the intent appears to be that there be a blank line before and after the `=== Global Setup ===` text, however if line 326 is executed, that intent is not preserved, because the output will go from:
```

=== Global Setup ===

```
to:
```

ERROR !!! winHTTP does not support using insecure mode.
=== Global Setup ==

```
which is likely not what the creator of line 333 intended.

Reply:
Won't matter anymore. Added support to winHttp.  We don't need the messages